### PR TITLE
TikTok documentation

### PR DESCRIPTION
**Pull request by ManikaAgarwalTikTok**
Change-Id: Ia3f64762137708d9e8105898daaee18102c45134

### CODE REVIEW

File: help/destinations/catalog/social/tiktok.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
```
Comment (vgiurgiu) [inline help/destinations/catalog/social/tiktok.md:1]: Comment about the image files: 
I see that you created a folder for them here: 
`help/destinations/catalog/social/tiktok/tiktok-ads-manager-ID.png`

However, the correct folder location should be: 
`help/destinations/assets/catalog/social/tiktok/tiktok-ads-manager-ID.png`
Notice the **assets** in the folder structure just above.

File: help/destinations/catalog/social/tiktok.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Before you are able to activate data to TikTok, you must authenticate with a TikTok Ads Manager account that you want to add an audience to. Details on sharing advertiser accounts can be found [in the TikTok API documentation](https://ads.tiktok.com/marketing_api/docs?id=1738373141733378). Specifically, you would need to give Adobe the "Audience Management" permission.
+
+## Supported identities {#supported-identities}
+
+TikTok supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+|GAID|Google Advertising ID|Select the GAID target identity when your source identity is a GAID namespace.|
+|IDFA|Apple ID for Advertisers|Select the IDFA target identity when your source identity is an IDFA namespace.|
```
Comment (vgiurgiu) [inline help/destinations/catalog/social/tiktok.md:35]: Question: I noticed a difference between the identities you list in the documentation and the ones in the destination configuration that you submitted: 

```
 "identityNamespaces": {
    "GAID": {
      "required": false,
      "acceptsAttributes": true,
      "acceptsCustomNamespaces": false,
      "transformation": "sha256(lower($))",
      "acceptedGlobalNamespaces": {
        "GAID": {
          "requiredTransformation": "sha256(lower($))"
        }
      }
    },
    "AAID": {
      "required": false,
      "acceptsAttributes": true,
      "acceptsCustomNamespaces": false,
      "transformation": "sha256(lower($))",
      "acceptedGlobalNamespaces": {
        "AAID": {
          "requiredTransformation": "sha256(lower($))"
        }
      }
    },
    "phone_number": {
      "required": false,
      "acceptsAttributes": true,
      "acceptsCustomNamespaces": false,
      "transformation": "sha256($)",
      "acceptedGlobalNamespaces": {
        "Phone_E.164": {
          "requiredTransformation": "sha256($)"
        },
        "Phone_SHA256_E.164": {}
      }
    },
    "email": {
      "required": false,
      "acceptsAttributes": true,
      "acceptsCustomNamespaces": false,
      "transformation": "sha256(lower($))",
      "acceptedGlobalNamespaces": {
        "Email": {
          "requiredTransformation": "sha256(lower($))"
        },
        "Email_LC_SHA256": {}
      }
    }
```

To be exact, I see IDFA here but AAID in the configuration. Not sure if this is on purpose or not, but wanted to call that out.

**Review by vgiurgiu — COMMENTED** [review]
@ManikaAgarwalTikTok - thank you for the updates, I have two more comments for you to please take a look at. Let me know if you have any questions.

File: help/destinations/catalog/social/tiktok.md (hunk 2 above)
Comment (ManikaAgarwalTikTok) [reply help/destinations/catalog/social/tiktok.md:35]: I've changed this to AAID in the documentation too. Let me know if it's ok now.

**Pull request merged**